src/components/Product.js

```diff
@@ -2,17 +2,14 @@ import React from 'react';
 import { useParams } from 'react-router-dom';
 
 const Product = (props) => {
-    const {productList} = props
-    const {productId} = useParams();
-    const product = productList.find(singleProduct => Number(productId) === singleProduct.id);
+    // const {productList} = props
+    // const {productId} = useParams();
+    // const product = productList.find(singleProduct => Number(productId) === singleProduct.id);
 
 
     return (
         <div>
             <h1>Product</h1>
-            {
```

Comment:
A bit worried about leaving this conditional out, will it cause a render issue? What are your thoughts, @zachgarrett24 ?